### ISSUE TITLE
could not list plans: data is not a valid PlanSpecsDrivesInnerCategory

### ISSUE THREAD

**Issue by displague** [StackPointCloud/packet-cli#474]
### What happened?

I can no longer list plans:
```
% metal plan list 
Error: could not list plans: data is not a valid PlanSpecsDrivesInnerCategory
```

### How can we reproduce it?

```
metal plan list 
```

CLI version (`metal --version`):

```
% metal -v
metal version v0.23.0
```

**Comment by ctreatma** [comment]
Could you re-run the command with `PACKNGO_DEBUG=true` to enable HTTP logging and paste the API response body (no headers) in a comment or in the issue description?

**Comment by ctreatma** [comment]
Fixed by #472, released in v0.23.1
